test/extended/fixtures/Dockerfile-jenkins-test-new-plugin

```diff
@@ -0,0 +1,6 @@
+FROM openshift/jenkins-1-centos7
+USER root
+RUN touch /opt/openshift/configuration/plugins.txt
+COPY ./jpi /opt/openshift/plugins
+RUN ls -la /opt/openshift/plugins
```

Comment:
yep the `ls` was just for debug;  i can remove it.

as far as one layer, yes, I don't see why not; though the squash would be in scripting after the docker build, and not
within the Dockerfile, correct?